<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-4IH7B69" ignored="0" name="Default test" tests="3" failures="0" timestamp="17 Oct 2020 18:02:52 GMT" time="0.029" errors="0">
  <testcase name="@BeforeMethod setup" time="0.004" classname="com.Amazon.qa.testcases.LoginpageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.Amazon.qa.base.TestBase.initialization(TestBase.java:42)
	at com.Amazon.qa.testcases.LoginpageTest.setup(LoginpageTest.java:27)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:599)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeMethod setup -->
  <testcase name="@BeforeMethod setup" time="0.0" classname="com.Amazon.qa.testcases.LoginpageTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod setup -->
  <testcase name="@BeforeMethod setup" time="0.0" classname="com.Amazon.qa.testcases.LoginpageTest">
    <skipped/>
  </testcase> <!-- @BeforeMethod setup -->
  <testcase name="loginPageTitleTest" time="0.0" classname="com.Amazon.qa.testcases.LoginpageTest">
    <skipped/>
  </testcase> <!-- loginPageTitleTest -->
  <testcase name="crmLogoImageTest" time="0.0" classname="com.Amazon.qa.testcases.LoginpageTest">
    <skipped/>
  </testcase> <!-- crmLogoImageTest -->
  <testcase name="LoginTest" time="0.0" classname="com.Amazon.qa.testcases.LoginpageTest">
    <skipped/>
  </testcase> <!-- LoginTest -->
</testsuite> <!-- Default test -->
